src/components/EnterpriseApp/EnterpriseApp.test.jsx

```diff
@@ -63,10 +62,16 @@ describe('<EnterpriseApp />', () => {
     toggleSidebarToggle: jest.fn(),
     loading: false,
     enableLearnerPortal: true,
-    enterpriseId: 'uuid',
-    enterpriseName: 'test-enterprise',
   };
 
+  basicProps.enterpriseId = 'uuid';
+  basicProps.enterpriseName = 'test-enterprise';
+
+  const invalidEnterpriseNameProps = {
+    ...basicProps,
+    enterpriseId: null,
```

Comment:
nit/question: if the creation of invalidEnterpriseNameProps overwrites the values of `enterpriseId` and `enterpriseName`, is it necessary to pull out and re-add those 2 values out of basicProps?

Reply:
I second Chris' nit/question and raise a third nit 😉 If we don't have the enterprise ID, we likely also don't have the enterprise name so it could be confusing to have `null` `enterpriseId` but a valid `enterpriseName`.